src/components/organisms/NavBarSchedule/calendar.ts

```diff
@@ -0,0 +1,42 @@
+import { eventEndTime, eventStartTime } from "utils/event";
+import { PersonalizedVenueEvent } from "types/venues";
+import { VenueEvent } from "types/venues";
+import { WithVenueId } from "utils/id";
+
+import { ics } from "./ics";
+
+export const createCalendar = (
+  allEvents: PersonalizedVenueEvent[] | WithVenueId<VenueEvent>[],
+  filename: string
+) => {
+  const cal = ics();
+  if (!cal) return;
+  allEvents.map((event: PersonalizedVenueEvent | WithVenueId<VenueEvent>) => {
+    return cal.addEvent(
+      event.id!,
+      event.name,
+      event.host || "",
```

Comment:
We default with `??` not `||`.

```suggestion
      event.host ?? "",
```